### PR TITLE
Integrate spee.ch link sharing from LBRY app content/channels

### PR DESCRIPTION
**Pull request by daovist**
#1028 @tzarebczan 

lbry speech share

lbry speech share 2

### CODE REVIEW

**Comment by kauffj** [comment]
@liamcardenas @tzarebczan apologies for being late to the party on this one, but I have some questions about this I'd like to discuss on next stand up before merging. Nothing related to @daovist's implementation (which I also didn't really look at very closely).

**Comment by liamcardenas** [comment]
@daovist can this say "View on Web" instead and have a globe icon?

Thank you!

**Comment by kauffj** [comment]
@daovist also only if price is 0, and also should probably go to URLs that include claim ids.

**Comment by daovist** [comment]
I can move the `View on Web` link on `<FilePage>` up by the `Subscribe` button like it is on `<ChannelPage>` if that is preferred

also, I moved the `<FilePage>` icon to the right since the screenshots above were taken. I can move it back to the left like the other icons if preferred

File: src/renderer/component/fileActions/view.jsx [hunk 1]
```diff
@@ -31,6 +33,15 @@ class FileActions extends React.PureComponent {
             label={__('report')}
           />
         )}
+        {speechSharable && (
+          <Link
+            button="text"
+            iconRight="icon-globe"
+            href={`https://spee.ch/${fileInfo.claim_name}#${claimId}`}
```
[the remark this answers is not in the document]
Comment (daovist) [reply src/renderer/component/fileActions/view.jsx]: Yeah. I'm all messed up from the redesign. I get hundreds of issues trying to merge.

**Pull request closed without merging**